Lib/http/server.py

```diff
@@ -678,8 +679,10 @@ def send_head(self):
         None, in which case the caller has nothing further to do.
 
         """
+        self.path = re.sub(r'(^(/|\\)+)', '/', self.path)
```

Comment:
Why not translating the path in parse_request()?

Why do you replace \\? Only "/" matters in URLs, especially for https://bugs.python.org/issue43223 issue.

Please add a comment explaining why you replace // with / and mention "bpo-43223" in your comment.

Reply:
Ok I'll be replacing this into the parse_request() and will request it.

Well I replaced it because there would be no sense to have a `\` in the beginning of the path and sometimes they also get converted to `/` automatically which may pose a security risk. Please correct me if I'm wrong.

bpo-43223: The reason is to provide protection against the open redirect attack which can be triggered due to the presence of  `//` at the beginning of the path.

Reply:
I just updated the commit. Please review.